pkg/configv2/default.go

```diff
@@ -0,0 +1,92 @@
+package configv2
+
+import (
+	"runtime"
+	"time"
+
+	"github.com/bacalhau-project/bacalhau/pkg/configv2/types"
+)
+
+var Default = types.Bacalhau{
+	Repo: "~/.bacalhau",
+	Name: "",
+	Client: types.Client{
+		Address: "http://0.0.0.0:1234",
+	},
+	Server: types.Server{
+		Address: "0.0.0.0",
+		Port:    1234,
+		TLS:     types.TLS{},
+		Auth: types.AuthConfig{
+			Methods: map[string]types.AuthenticatorConfig{
+				"ClientKey": {
+					Type: "challenge",
+				},
+			},
+		},
+	},
+	Orchestrator: types.Orchestrator{
+		Enabled:   true,
+		Listen:    "0.0.0.0",
+		Port:      4222,
+		Advertise: "0.0.0.0",
+		NodeManager: types.NodeManager{
+			DisconnectTimeout: types.Duration(time.Second * 30),
+			AutoApprove:       true,
+		},
+		Scheduler: types.Scheduler{
+			Workers:              runtime.NumCPU(),
+			HousekeepingInterval: types.Duration(time.Second * 30),
+			HousekeepingTimeout:  types.Duration(time.Minute * 2),
+		},
+		Broker: types.EvaluationBroker{
+			VisibilityTimeout: types.Duration(time.Minute),
+			MaxRetries:        10,
```

Comment:
Iff we want to add that, field `RetryTimeout` would be a clearer as it implies a timeout related to the retry process - the amount of time to wait during each retry. Multiplying `MaxRetries` by `RetryTimeout` could conceptually give you a `MaxRetryDuration`

`MaxRetryDuration` introduces ambiguity: It could imply the total time allowed for all retries, the maximum time a single retry attempt should take, or even the maximum time the retry logic should keep trying overall.